src/Components/Auth/Login/login.js

```diff
@@ -0,0 +1,88 @@
+import { useState, useEffect } from "react";
+import { useAuth } from "../../../Context/authContext";
+import { useNavigate, useLocation } from "react-router-dom";
+import "../Login/login.css";
+const Login = () => {
+  const { loginAuth, token } = useAuth();
+  const [loginForm, setLoginForm] = useState({
+    email: "",
+    password: "",
+  });
+  useEffect(() => {
+    loginAuth(loginForm.email, loginForm.password)
+  }, [loginForm.email, loginForm.password]);
+  const navigate = useNavigate();
+  const location = useLocation();
+  if (token) navigate(location?.state?.from || "/", { replace: true });
+  const loginHandler = ()=>{
+      setLoginForm ((loginForm)=> ({
+          ...loginForm,
+          email:"rohandubey154@gmail.com",
+          password:"rohan12345"
+      }))
+  }
+  return (
+    <div className="wrapper">
+      <div className="container-login">
+        <div className="col-left">
+          <div className="login-text">
+            <h2>Welcome Back</h2>
+            <p>Create your account now. </p>
+            <a className="btn" href="">
+              Sign Up
+            </a>
+          </div>
+        </div>
+        <div className="col-right">
+          <div className="login-form">
+            <h2>Login</h2>
+            <form onSubmit={(e)=> e.preventDefault()}>
+              <p>
+                <label>
+                  email address<span>*</span>
+                </label>
+                <input
+                  type="email"
+                  placeholder="naruto@naruto.com"
+                  value={loginForm.email}
+                  onChange={(e) =>
+                    setLoginForm((loginForm) => ({
+                      ...loginForm,
```

Comment:
don't do things here, create a function and do things there.
If something is only of one line then do it in here.